app/utils/rxjs/RxjsPeer.client.ts

```diff
@@ -166,13 +166,17 @@ export class RxjsPeer {
 	async createSession(peerConnection: RTCPeerConnection) {
 		console.debug('🆕 creating new session')
 		const { apiBase } = this.config
-		// create an offer and set it as the local description
-		await peerConnection.setLocalDescription(await peerConnection.createOffer())
+		// create an offer
+		const offer = await peerConnection.createOffer()
+		// Turn on Opus DTX to save bandwidth
+		offer.sdp = offer.sdp!.replace('useinbandfec=1', 'usedtx=1;useinbandfec=1')
```

Comment:
```suggestion
		offer.sdp = offer.sdp?.replace('useinbandfec=1', 'usedtx=1;useinbandfec=1')
```
Def not a blocker, and this will probably never be `undefined` in this case, but would it make sense to use the optional chaining operator here?

Reply:
Yes makes sense. Fixed.